notes.md

```diff
@@ -0,0 +1,10 @@
+- Making tab active border overlap the ribbon border. This is typically how active border indicators work
```

Comment:
We shouldn't commit this file, right?

Reply:
I think this one was committed by @dbanksdesign, I'm happy to remove it though.